alica_engine/include/engine/syncmodule/SyncRow.h

```diff
@@ -14,23 +14,21 @@ struct SyncData;
 class SyncRow
 {
 public:
-    SyncRow();
-    SyncRow(const SyncData& sd);
+    SyncRow() = delete;
+    explicit SyncRow(const SyncData& sd);
     virtual ~SyncRow();
```

Comment:
I make destructors virtual by default. That way, I don't run into problems when I add a derived class later on. Should we check all classes, whether the have some derived class and remove the virtual keyword, if they don't have one? 
Otherwise, I don't get the point. :sweat_smile: